pkg/logstash.sysv

```diff
@@ -96,12 +97,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      if [ $KILL_ON_STOP_TIMEOUT -eq 1 ] ; then
+      if [ $LS_KILL_ON_STOP_TIMEOUT -eq 1 ] ; then
```

Comment:
Quoting it doesn't make it any more a string than it already is ;)

I misspoke. I was thinking of `=` which has unexpected behavior (error) when you do not quote. It's still true of `-eq` though you always get an error (even if the error is different) if you don't have something coercible into a number:

```
$ [ $FOO = 1 ]
sh: 1: [: =: unexpected operator
$ [ "$FOO" = 1 ]
$ [ $FOO -eq 1 ]
sh: 3: [: -eq: unexpected operator
$ [ "$FOO" -eq 1 ]
sh: 4: [: Illegal number:
```

Still, we should quote this variable so the error is most informative "illegal number" instead of the very confusing "unexpected operator"